README_zh.md

```
@@ -0,0 +1,188 @@
+- 🌈 **主题定制**：灵活的主题系统，支持多种主题和自定义样式
+- 📦 **Tree Shaking**：针对 Tree Shaking 优化，按需导入
+- 🔄 **流式支持**：内置流式 AI 响应支持
+- 💾 **存储策略**：灵活的存储策略（LocalStorage、IndexedDB、自定义）
+
+## 📦 包说明
+
+TinyRobot 是一个 monorepo，包含以下包：
+
+| 包                                                      | 说明                                           | 版本                                                                                                                       |
+| ------------------------------------------------------- | ---------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| [`@opentiny/tiny-robot`](./packages/components)         | 核心组件库，包含所有 AI 交互组件               | [![npm](https://img.shields.io/npm/v/@opentiny/tiny-robot.svg)](https://www.npmjs.com/package/@opentiny/tiny-robot)      |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)